.github/workflows/go.yml

```diff
@@ -10,6 +10,9 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            image: [ "mongo:3.6", "mongo:4.0", "mongo:4.2", "percona/percona-server-mongodb:3.6", "percona/percona-server-mongodb:4.0" ]
```

Comment:
I would suggest to keep each version in separate line.

Reply:
This ^